src/content/docs/en/guides/syntax-highlighting.mdx

```diff
@@ -111,9 +111,12 @@ export default defineConfig({
 
 ### Customizing Shiki themes
 
-You can follow [Shiki's own theme documentation](https://shiki.style/themes) for more customization options for themes, light vs dark mode toggles, or styling via CSS variables.
+You can follow [Shiki's own theme documentation](https://shiki.style/themes) for more customization options for themes, [light vs dark mode toggles](https://shiki.style/guide/dual-themes), or styling via [CSS variables](https://shiki.style/guide/theme-colors#css-variables-theme).
 
-Astro code blocks are styled using the `.astro-code` class, so you will need to replace the `.shiki` class in the examples with `.astro-code`.
+Astro code blocks have some differences compared to the default Shiki documentation:
```

Comment:
```suggestion
You will need to adjust the examples from Shiki's documentation for your Astro project by making the following substitutions:
```
I do like the particular reminder to change/replace that the original wording had. What about something like this?

Reply:
Oh, really liking the suggestion, thanks a lot :raised_hands: